packages/prover/src/utils/process.ts

```diff
@@ -24,22 +24,42 @@ export const supportedELRequests: Record<string, ELVerifiedRequestHandler<any, a
   eth_estimateGas: eth_estimateGas,
 };
 
-export function splitRequestsInChunks(payload: JsonRpcRequestOrBatch): {
+export const verifiableMethods = Object.keys(verifiableMethodHandlers);
+export const alwaysAllowedMethods = ["eth_subscribe", "eth_unsubscribe", "eth_getProof"];
```

Comment:
does it make sense to have an enum or defined typed for all the available methods?

Reply:
Don't think so enum is gonna help here. We are not gonna reuse these anywhere else as types.